facefusion/audio.py

```diff
@@ -3,14 +3,19 @@
 import numpy
 import scipy
 
+import facefusion.globals
 from facefusion.filesystem import is_audio
 from facefusion.ffmpeg import read_audio_buffer
 from facefusion.typing import Fps, Audio, Spectrogram, AudioFrame
+from facefusion.audio_extractor import batch_extract_voice
 
 
 def get_audio_frame(audio_path : str, fps : Fps, frame_number : int = 0) -> Optional[AudioFrame]:
 	if is_audio(audio_path):
-		audio_frames = read_static_audio(audio_path, fps)
+		if facefusion.globals.audio_extract_voice:
```

Comment:
this is not good - you changed the behaviour of the get_audio_frame by a global rather than introducing a new method - this is the same like having a flag, what I already mentioned is a bad practice